keyboards/mt84/mt84.h

```diff
@@ -0,0 +1,35 @@
+   /* Copyright 2020 MaiKong<704340378@qq.com>
+  * 
+  * This program is free software: you can redistribute it and/or modify 
+  * it under the terms of the GNU General Public License as published by 
+  * the Free Software Foundation, either version 2 of the License, or 
+  * (at your option) any later version. 
+  * 
+  * This program is distributed in the hope that it will be useful, 
+  * but WITHOUT ANY WARRANTY; without even the implied warranty of 
+  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the 
+  * GNU General Public License for more details. 
+  * 
+  * You should have received a copy of the GNU General Public License 
+  * along with this program.  If not, see <http://www.gnu.org/licenses/>. 
+  */
+#pragma once
+
+#include "quantum.h"
+
+#define LAYOUT( \
```

Comment:
This keyboard's layout matches the 75_ansi [Community Layout](https://docs.qmk.fm/#/feature_layouts).

```suggestion
#define LAYOUT_75_ansi( \
```